gradle.properties

```
@@ -1 +1,2 @@
 org.gradle.caching=true
+version=0.0.1-SNAPSHOT
```

Let's make it a bit more "default"-ish :)

```suggestion
version=0-SNAPSHOT
```

---

tried, doesn't work - not valid for semver parser in pf4j